### ISSUE TITLE
Set up a new documentation website

### ISSUE THREAD

**Issue by bob2517** [Active-CSS/active-css#284]
The current documentation website is getting a bit large, and it's way too wordy. It's quick to find things, but you have to know where to go. And there are techniques for doing things which are only implied if you know the full syntax. New documentation methods are desperately needed. The ACSS behind it is version 1 and could do with an upgrade. The HTML and CSS on the site is not very clean or clever. There are no fancy page transitions, so we're missing a trick on demonstrating ACSS's inherent and largely under-documented SPA capabilities. It doesn't have a proper pro feel to it.

Make a new website that makes it simpler for new and existing devs to grasp concepts. Involve a pro graphic designer (not a web designer) with prototyping and do a number on it just like a work project. Add any necessary animation, etc., to make concepts clearer. Keep it on self-hosted servers so full flexibility and ownership in kept intact.

**Comment by bob2517** [comment]
That's very kind of you, but I've already agreed terms with a graphic designer from my workplace.

I do need to get the code editor re-designed though, and that was not part of the deal with this guy. Would that be something you would be interested in helping on? If not, that's ok. The current design of it is very blocky and I've seen better ones around, but I'm no designer :) It's not an immediate thing though - I probably won't be starting on this for a couple of months.

**Comment by dragontheory** [comment]
Absolutely! Anything I can do to help. I will start the "due diligence" research and look around to see what others are doing such as MDN and like.

**Comment by bob2517** [comment]
That would be awesome, thank you! :)
